package MazeSolve;

import java.io.IOException;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		PlayerDirection8 p = new PlayerDirection8();
		int reset = 1;
		Maze.getInstance().cursor(1, 1);
	
		//while(reset == 1){
		for (int i = 0; i<9;i++){
			while(Maze.getInstance().getMap(8,8) == 0){
				Maze.getInstance().cursor(i, i);
				p.start();
				Maze.getInstance().showMaze();
				p.showPathCount();
				System.out.println(p.location_y);
				delayOneSec();
				//screenClear();
				
			}
			System.out.print("Maze is solved. try again? (1=yes/2=no)");
			Scanner sc = new Scanner(System.in);
			reset = sc.nextInt();
		}
		
	}
	
	public static void delayOneSec(){
		try {
		    Thread.sleep(1000);                 
		} catch(InterruptedException ex) {
		    Thread.currentThread().interrupt();
		}
	}
	
	public static void screenClear(){
		try {
			Runtime.getRuntime().exec("clear");
		} catch (IOException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
	}

}
